react.js

```diff
@@ -19,17 +19,17 @@ module.exports = {
     'jsx-quotes': [ error, 'prefer-double' ],
     'react/jsx-indent': [ warn, 2 ],
     'react/jsx-indent-props': [ warn, 2 ],
-    'react/forbid-prop-types': [ error, { 'forbid': [ any ] } ],
+    'react/forbid-prop-types': [ error, { forbid: [ any ] } ],
     'react/jsx-boolean-value': [ off ],
     'react/sort-comp': [ error, {
-      'order': [
+      order: [
```

Comment:
I can't say I am a fan of this, to be perfectly honest.

If I understand this [rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-comp.md) right, this would enforce this order `displayName, propTypes, contextTypes, ...`

We would have to refactor almost all of our components ( in the `dsx` project at least ) because we don't use propTypes that way. We use it like this `MyComponent.propTypes = { ... }`

Moreover, with this rule, `propTypes` and `defaultProps` would have to be declared before the `constructor`, which I would not like. I believe the `constructor` should be declared first.

I only took a quick glance at the rule, but if it behaves the way I think, I wouldn't want this merged.

Reply:
I haven't changed that, just the quotes around it

Reply:
Does this mean that the rule isn't working properly anyway? Seeing as `dsx` should be using `eslint-config`. Out of scope for this PR but worth looking into.

Reply:
Ahh ok, I see. I just saw it there, looked it up and saw what it did. I think the rule works correctly, I'm not sure, but perhaps it only enforces the order if `propTypes`, etc are declared inside and doesn't apply to them if declared outside.